### PR TITLE
⚡ Fix compatibility with Windows 

### PR DESCRIPTION
**Pull request by schellingerhout**
Windows can't find some of the `utils.run` commands or expects `\` instead `/` in paths. This means that Windows users can't create sections or new posts using the client. The problem also exists if running from Git Bash.  The solution is to replace the shell commands with `fs` calls that know how to handle Windows.

### CODE REVIEW

File: src/flow.js [hunk 1]
```diff
@@ -516,13 +516,9 @@ export default class flow {
     }
 
     if (!utils.directoryExists('./assets/')) {
-      utils.run('mkdir assets', false)
-        .then(() => {
-          processChanges()
-        });
-    } else {
-      processChanges()
+       utils.directoryCreate('assets');
```
Comment (schellingerhout) [inline src/flow.js:519]: I could have dropped the `if` as well because `utils.directoryCreate` already does that too

**Comment by schellingerhout** [comment]
I just realized that Contributing said to use the development branch. I will retarget

File: src/utils.js [hunk 2]
```diff
@@ -56,6 +56,38 @@ export default class utils {
     });
   }
 
+  static normalizePath(filePath) {
```
Comment (schellingerhout) [inline src/utils.js:59]: Addresses #66

**Comment by schellingerhout** [comment]
I added some more work to address #66. In particular I handled the quirks from Windows by using `fs.normalize`, but also in case of spaces in the filename removing single and double quotes only if they are at the front and back. Powershell also adds an ampersand and space when quoting.

**Comment by nunocoracao** [comment]
@schellingerhout you are a legend :)
I really appreciate this... Will merge and test it directly in main before making a new release

**Pull request merged**